pypesto/visualize/profiles.py

```diff
@@ -38,8 +38,8 @@ def profiles(results, ax=None, profile_indices=None, size=(18.5, 6.5),
         Labels for line plots, one label per result object.
     x_labels: list of str
         Labels for parameter value axes (e.g. parameter names).
-    profile_list_id: int, optional
-        Index of the profile list to be used for profiling.
+    profile_list_ids: int or list of ints, optional
```

Comment:
In general, it would be good if we can replace as many as possible in-docstring-typehints by real, testable, typehints. But fine here for the moment ...